src/DistributedWriteAheadLog/KafkaWALConsumer.h

```diff
@@ -49,6 +49,8 @@ class KafkaWALConsumer final : private boost::noncopyable
     /// Commit offset for a partition of a topic
     int32_t commit(const TopicPartitionOffsets & tpos);
 
+    TopicPartitionStatsPtr getTopicPartitionStats(const TopicPartitionOffset & tpo) const;
```

Comment:
change 'tpo' to offset, such abbreviation is not consistant with Clickhouse's style.

Reply:
yeah, but in the whole file, we should have consistent style with others, such as `const TopicPartitionOffsets & tpos` or `const TopicPartitionOffset & tpo` .

Reply:
Yes， all these variables should be changed accordingly